Obligatorio/MinTurFrontend/src/app/core/navbar/navbar.component.html

```diff
@@ -19,6 +19,11 @@
       </li>
     </div>
     <div *ngIf="IsLoggedIn()">
+      <li class="my-2 md:my-0">
+        <a routerLink="explore/" [routerLinkActive]="['text-purple']" routerLinkActiveOptions="{exact: false}" class="block py-1 md:py-3 pl-1 align-middle text-gray-600 no-underline hover:text-purple hover:font-semibold">
+            <i class="far fa-map mr-3"></i><span class="w-full inline-block pb-1 md:pb-0 text-sm">Explorar</span>
+        </a>
+    </li>
```

Comment:
Bien resuelto, quizas en un futuro estaria bueno hacer un ng-for  y ahorrar codigo